.github/workflows/monitor-usns.yml

```diff
@@ -1,8 +1,9 @@
-name: Snaps
+name: Monitor USNs
```

Comment:
Forgive my ignorance, but what does `USN` mean here?

Reply:
Sorry, [Ubuntu Security Notices](https://ubuntu.com/security/notices). I'll add it in a comment.